modules/ossm-kiali-viewing-logs.adoc

```diff
@@ -0,0 +1,35 @@
+////
+Module included in the following assemblies:
+* service_mesh/v2x/ossm-observability.adoc
+////
+
+[id="ossm-viewing-logs_{context}"]
+= Viewing logs in the Kiali console
+
+You can view logs for your workloads in the Kiali console.  The *Workload Detail* page includes a *Logs* tab which displays a unified logs view that displays both application and proxy logs. You can select how often you want the log display in Kiali to be refreshed.
+
+To change the logging level on the logs displayed in Kiali, you change the logging configuration for the workload or the proxy.
+
+// TODO once released - Kiali can also add-in trace span information to help identify important traces based on associated logging. More powerful features include substring or regex Show/Hide, full-screen, and the ability to set proxy log level without a pod restart.
```

Comment:
Just pointing out this TODO section in case you want to revisit it.

Reply:
This was from the upstream docs and won't be released in the product until our next release.  But thanks!